android/quest/build.gradle.kts

```diff
@@ -240,6 +240,12 @@ android {
       versionNameSuffix = "-cadre"
       manifestPlaceholders["appLabel"] = "Cadre App"
     }
+    create("sideir") {
+      dimension = "apps"
+      applicationIdSuffix = ".sideir"
+      versionNameSuffix = "-sideir"
+      manifestPlaceholders["appLabel"] = "SID EIR"
+    }
```

Comment:
if we're prefixing this with `sid`, should we prefix the `cadre` and `bunda` app with sid?

thinking about it, the specificity here might be more to Indonesia vs SID, what about?

- `id-bunda`
- `id-cadre`
- `id-eir`

cc @MargaretNjenga @f-odhiambo

Reply:
I agree to rename all of them.